packages/components/Button/src/Button.icon.ts

```diff
@@ -0,0 +1,22 @@
+import { Image } from 'react-native';
+import { RasterImageIconProps, IconProps } from '@fluentui-react-native/icon';
```

Comment:
Did we not have icon support on the original FURN button before?

Reply:
Yep icons are already supported, these are just the imports in the separated common code.